src/PhpSpreadsheet/Writer/Xlsx/StringTable.php

```diff
@@ -97,7 +97,7 @@ public function writeStringTable(array $stringTable)
                 if ($textToWrite !== trim($textToWrite)) {
                     $objWriter->writeAttribute('xml:space', 'preserve');
                 }
-                $objWriter->writeRawData($textToWrite);
+                $objWriter->writeRawData(StringHelper::applyCharsLimit($textToWrite));
```

Comment:
Should this be applied on write? Or when the cell value is set? e.g. perhaps through the value binder